lang/en-gb.xml

```diff
@@ -553,7 +553,7 @@ You can earn trophies by using an application that supports the trophy feature.<
 		<delete_user>Delete User</delete_user>
 		<user_delete>Select the user you want to delete.</user_delete>
 		<user_delete_msg>The following user will be deleted.</user_delete_msg>
-		<user_delete_message>If you delete the user, that user's saved data, trophies will be deleted.</user_delete_message>
+		<user_delete_message>If you delete the user, that user's saved data and trophies will be deleted.</user_delete_message>
```

Comment:
Revert

Reply:
This is what it looks like without. 
![image](https://user-images.githubusercontent.com/69767396/230465263-a299c163-41c2-4c2f-92a8-4d5cd7a40904.png)
it doesn't make sense.

Reply:
Is ps4 screen, is good

Reply:
I don't understand...

Reply:
This string is from ps4 interface for create user

Reply:
But it doesn't make sense.

Reply:
fixed